packages/deprecation-crawler/src/lib/cli/index.ts

```diff
@@ -25,10 +25,6 @@ export function runCli(
 ) {
   const yargs = setupYargs(cliCfg.commands, cliCfg.options);
   const argv = yargs.argv;
-  const isDefaultCommand = argv._[0] === undefined;
-  if (isDefaultCommand) {
-    cliCfg.commands
-      .find((c) => c.command === defaultCommand)
-      .module.handler(argv);
-  }
+  const command = argv._[0] || defaultCommand;
```

Comment:
I had to do this to be able to run the "group" or "history" command.
But now I also get the error (but it's not blocking)

```
YError: Invalid second argument. Expected object but received function.
    at argumentTypeError (C:\Users\tdeschryver\dev\libs\deprecation-manager\node_modules\yargs\build\lib\argsert.js:64:11)
    at C:\Users\tdeschryver\dev\libs\deprecation-manager\node_modules\yargs\build\lib\argsert.js:45:17
    at Array.forEach (<anonymous>)
    at Object.argsert (C:\Users\tdeschryver\dev\libs\deprecation-manager\node_modules\yargs\build\lib\argsert.js:38:25)
    at Object.option (C:\Users\tdeschryver\dev\libs\deprecation-manager\node_modules\yargs\build\lib\yargs.js:594:19)
    at C:\Users\tdeschryver\dev\libs\deprecation-manager\node_modules\yargs\build\lib\command.js:168:28
    at Array.forEach (<anonymous>)
    at Object.runCommand (C:\Users\tdeschryver\dev\libs\deprecation-manager\node_modules\yargs\build\lib\command.js:167:49)
    at Object.parseArgs [as _parseArgs] (C:\Users\tdeschryver\dev\libs\deprecation-manager\node_modules\yargs\build\lib\yargs.js:1043:55)
    at Function.get [as argv] (C:\Users\tdeschryver\dev\libs\deprecation-manager\node_modules\yargs\build\lib\yargs.js:986:25)
```